rdflib/compare.py

```diff
@@ -290,7 +292,7 @@ def _initial_color(self):
 
     def _individuate(self, color, individual):
         new_color = list(color.color)
-        new_color.append((len(color.nodes)))
+        new_color.append(tuple([len(color.nodes)]))
```

Comment:
I wouldn't object to the change. I couldn't remember the syntax for a single-element tuple.